static_src/actions/user_actions.js

```diff
@@ -201,19 +201,18 @@ const userActions = {
       { space_users: { singular: 'space' } },
       { org_users: { singular: 'organization' } }
     );
+    const entity = viewTypeNouns[currentViewedType].singular;
 
     if (verified) {
-      description = `The account for ${email} is now associated to this ` +
-        `${viewTypeNouns[currentViewedType].singular}. Control their ` +
-        `${viewTypeNouns[currentViewedType].singular} roles below.`;
+      description = `The cloud.gov account for ${email} is now associated to this ` +
+        `${entity}. Control their ${entity} roles below.`;
     } else {
-      description = `There was no cloud.gov account found for ${email} ` +
-        'or the user has not verified their account by logging in.' +
-        'They have been sent an email cloud.gov invitation. Their account ' +
-        `has been associated to this ${viewTypeNouns[currentViewedType].singular}` +
-        ` and their ${viewTypeNouns[currentViewedType].singular}` +
-        ' roles can be controlled below.';
+      description = `No verified cloud.gov account was found for ${email}, so ` +
```

Comment:
I'm wondering if the typical user will know what a `verified` account means. I can see just starting off the description with `The user has been sent...`